### ISSUE TITLE
[IntelliJ-ThirdParty][A11yMAS][Accessibility][ColourContrast][MAS1.4.3][IntelliJ][Wipro]: Colour contrast ratio is below 4.5:1 for light text in "Browse Repositories" dialog.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1376]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. IntelliJ IDEA start screen appears, Navigate and select  Plugins under Configure combo box. 
3. Plugins dialog will open.
4. Select Browse repositories  button.
5. In Browse Repositories dialog observe colour contrast ratio for light text.
6. Same issue is observed for all the light text present in Browse Repositories Window

**Expected Result:**
Colour contrast ratio should be greater than or equal to 4.5:1 for light text.

**Actual Result:**
Colour contrast ratio is 3.8:1 for light text in window

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)
**CCA version**: 2.5.0.0

![cca for lite text in plugins and browse repositories](https://user-images.githubusercontent.com/22956130/35680982-d711e896-0710-11e8-82ed-dc0aea2c21e3.png)
.

**Comment by Eskibear** [comment]
It's IntelliJ IDEA's own dialog, out of the toolkit's scope.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749813
